pos_event_registration/static/description/index.html

```diff
@@ -0,0 +1,130 @@
+<section class="oe_container">
+  <div class="oe_row oe_spaced">
+    <div class="oe_span12">
+      <h2 class="oe_slogan" style="color:#875A7B;">Register Event Attendees</h2>
+      <h3 class="oe_slogan">Process Attendees and Sell Tickets via POS</h3>
+    </div>
+  </div>
+</section>
+
+<section class="oe_container">
+  <div class="oe_row oe_spaced">
+    <div class="oe_span12">
+      <p class="oe_mt32">
+	<div class="alert alert-warning" style="padding:0.3em 0.6em; font-size: 150%;">
+	</div>
+      </p>
+    </div>
+  </div>
+</section>
+
+<section class="oe_container oe_dark">
+  <div class="oe_row oe_spaced">
+    <h3 class="oe_slogan">How It Works</h3>
+    <div class="oe_span12 text-center">
+      <p class="oe_mt32">
+ 	<font style="font-size: 120%;">
+
+	</font>
+      </p>		
+    </div>
+    <div class="oe_screenshot" align="center">
+      <img style="max-width: 80%" src=""/>
+    </div>
+  </div>
+</section>
+
+<section class="oe_container">
+  <div class="oe_row oe_spaced">
+    <div class="oe_span12">
+      <p class="oe_mt32">
+ 	<font style="font-size: 120%;">
+	  <ul>
+	    <li>
+
```

Comment:
I do not see here any description of the module, just dots

Reply:
fixed, now you can see